enterprise_access/apps/subsidy_access_policy/models.py

```diff
@@ -317,16 +317,13 @@ def total_spend_limit_for_all_policies_associated_to_subsidy(self):
         """
         Sums the policies spend_limit excluding the db's instance of this policy
         """
-        policy_balances = []
-        policy_balances.append(self.spend_limit or 0)
-        sibling_policies = SubsidyAccessPolicy.objects.filter(
+        spend_limit = self.spend_limit or 0
```

Comment:
I know it's redundant to check self.active in this function because theoretically it's already covered by clean(), but it might be good to do it here too because there might be other future callers of this function besides clean().
```suggestion
        spend_limit = self.spend_limit if self.active and self.spend_limit or 0
```